keyboards/ploopyco/nano_2/readme.md

```diff
@@ -0,0 +1,34 @@
+# Ploopy Nano 2 Trackball
+
+![Ploopyco Trackball Nano](https://ploopy.co/wp-content/uploads/2021/06/2.jpg)
+
+It's a DIY, QMK Powered Trackball...Nano!
+
+* Maintainer: [PloopyCo](https://github.com/ploopyco)
+* Key contributors: [Drashna Jael're](https://github.com/drashna/), [Germ](https://github.com/germ/)
+* Hardware Supported: RP2040
+* Hardware Availability: [Store](https://ploopy.co/nano-2), [GitHub](https://github.com/ploopyco)
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb ploopyco/nano_2 -km default
```

Comment:
```suggestion
    make ploopyco/nano_2/rev2_003:default

Flashing example for this keyboard:

    make ploopyco/nano_2/rev2_003:default:flash
```

Reply:
Fixed in [50ef10a](https://github.com/qmk/qmk_firmware/pull/25762/commits/50ef10a2e88f643a6d1defc78b4c0d9c1acadece).